### ISSUE TITLE
Release 0.16.0

### ISSUE THREAD

**Issue by sebright** [census-instrumentation/opencensus-java#1419]
I'm planning to make a major release next week to release opencensus-contrib-log-correlation-log4j2.  This issue is for tracking anything that needs to be done before the release.

/cc @songy23 @bogdandrutu @g-easy

**Comment by bogdandrutu** [comment]
Please hold until we fix the Distribution buckets issue with OpenMetrics.

**Comment by sebright** [comment]
Which issue needs to be fixed?

**Comment by sebright** [comment]
I discussed this issue with @bogdandrutu, and we decided to exclude the metrics package from the next release.

**Comment by sebright** [comment]
See #1426

**Comment by sebright** [comment]
#1426 temporarily moved the metrics package out of opencensus-api, so I plan to make the release once the Log4j log correlation PRs are merged.

**Comment by sebright** [comment]
I'm planning to make the release soon, since the log correlation PR was merged.

**Comment by sebright** [comment]
The only remaining step of the release is updating the Bazel example.  I'm getting some errors and warnings when I run the bazel generate_workspace command: https://gist.github.com/sebright/dea70d1d07303b24fba84bf6f3e59eae

The example doesn't build with the generated file:

```
$ bazel clean && bazel build :all
INFO: Starting clean (this may take a while). Consider using --async if the clean takes more than several minutes.
INFO: Build options have changed, discarding analysis cache.
ERROR: Analysis of target '//:opencensus_examples' failed; build aborted: no such package '@com_fasterxml_jackson_core_jackson_core//jar': The repository could not be resolved
INFO: Elapsed time: 0.375s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (13 packages loaded)
    currently loading: @com_google_protobuf//
```

I also tried specifying a newer version of gRPC (1.13.1) in the generate_workspace command, but that also didn't work.

**Comment by songy23** [comment]
Could be due to `com_fasterxml_jackson_core_jackson_core` is no longer a transitive dependency for example. Could you try removing https://github.com/census-instrumentation/opencensus-java/blob/660e8f375bb483a3eb817940b3aa8534f86da314/examples/BUILD.bazel#L69 after updating the gRPC version?

**Comment by songy23** [comment]
This issue is superseded by https://github.com/census-instrumentation/opencensus-java/issues/1447.

**Comment by sebright** [comment]
@songy23 Thanks! The build succeeded after I made those two changes and added `--artifact=com.google.guava:guava-jdk5:23.0` to the generate_workspace command.

**Comment by sebright** [comment]
I actually forgot to upgrade gRPC when I ran the generate_workspace command, and I found that upgrading gRPC to 1.13.1 causes other errors.  I made a PR that only upgrades opencensus-java: https://github.com/census-instrumentation/opencensus-java/pull/1457
